common/views/components/WorkHeader/WorkHeader.js

```diff
@@ -71,11 +71,17 @@ const WorkHeader = ({ work }: Props) => {
                   [spacing({ s: 2 }, { margin: ['right'] })]: true,
                 })}
               >
-                <LinkLabels
-                  items={work.contributors.map(({ agent }) => ({
-                    text: agent.label,
-                    url: `/works?query="${agent.label}"`,
-                  }))}
+                <WorkTags
+                  tags={work.contributors.map(({ agent }) => {
+                    return {
+                      query: agent.label,
+                      textParts: [agent.label],
+                      linkAttributes: {
+                        href: `/works?query="${agent.label}"`,
```

Comment:
Why not use the `workLink`?

Reply:
Ah yeah – the `workLink` service is within the `works` app, but this component is in `common` – as I understand it, we can't then access the service from here?

Reply:
Yea, this is a question I am battling with the moment as to what is common and what isn't.

Personally I would say none of the works stuff is, and a lot of the content stuff isn't relevant to work.

It makes it easier to work with, but will, in the long run start to kill our build speed and options for optimising. Platform are going through this now.

It feels safer to keep this logic in 1 place, so maybe we just move the `workLinks` over?  Weirdly I would say they have way more potential to become `common` over the `WorkHeader`.